### PR TITLE
Initial AutoMM Document + Disable multi-gpu training in interactive environment

### PR DESCRIPTION
**Pull request by sxjscience**
*Issue #, if available:*

*Description of changes:*
- [x] Add the initial documentation about AutoMMPredictor
- [x] Force to use 1 gpu if trying multi-gpu in an interactive environment.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1858-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1858/2/index.html

File: multimodal/src/autogluon/multimodal/presets.py [hunk 1]
```diff
@@ -89,13 +89,13 @@ def get_basic_automm_config():
     }
 
 
-def get_automm_preset(preset: str):
+def get_automm_presets(presets: str):
```
Comment (zhiqiangdon) [inline multimodal/src/autogluon/multimodal/presets.py:92]: Do we really need to use the plural form of preset everywhere? I think it's better to keep the singular form in most places except for the one in `.fit()`. No need to propagate the wrong form to other places.

**Comment by szha** [comment]
Job PR-1858-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1858/5/index.html

**Review by zhiqiangdon — APPROVED** [review]
LGTM

**Comment by szha** [comment]
Job PR-1858-6 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1858/6/index.html

**Review by bryanyzhu — APPROVED** [review]
LGTM

**Pull request merged**